x-pack/plugins/infra/public/pages/metrics/inventory_view/components/bottom_drawer.tsx

```diff
@@ -74,9 +76,11 @@ export const BottomDrawer = ({ interval, formatter, view }: Props) => {
   }, [isOpen, trackDrawerOpen, changeTimelineOpen]);
 
   return view === 'table' ? (
-    <BottomPanel hasBorder={false} hasShadow={false} borderRadius="none" paddingSize="s">
-      <KubernetesButton />
-    </BottomPanel>
+    nodeType === 'pod' ? (
```

Comment:
**nit:**  it wasn't easy to grasp at first look with the double ternaries nested, maybe inverting the first check condition might make it easier to read  `view !== 'table'`? 
Not sure though since you will have to repeat the `view === 'table'` check

Reply:
I don't know if inverting would make it easier to read. Maybe If I extract the inner ternary if, though. Would it facilitate?

Reply:
This would even be better 👍